File: Backend/PortfolioAPI/Models/Entities/Certification.cs
namespace PortfolioAPI.Models;

public class Certification
{
    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string? NameAr { get; set; }
    public string? Issuer { get; set; }
    public string? IssuerAr { get; set; }
    public string? IssueDate { get; set; }
    public string? ExpiryDate { get; set; }
    public string? CredentialUrl { get; set; }
    public string? LogoUrl { get; set; }
    public string? ImageUrl1 { get; set; }
    public string? ImageUrl2 { get; set; }
    public string? ImageUrl3 { get; set; }
    public string Category { get; set; } = "Technical";
    public string? CategoryAr { get; set; }
    public int SortOrder { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: src/core/types/certification.ts
export interface Certification {
  id: number;
  name: string;
  nameAr?: string;
  issuer: string;
  issuerAr?: string;
  issueDate?: string;
  expiryDate?: string;
  credentialUrl?: string;
  logoUrl?: string;
  imageUrl1?: string;
  imageUrl2?: string;
  category?: string;
  categoryAr?: string;
  sortOrder: number;
}
